File: PRAK603-2210817210023-Muhammad Shofy Akmal.py
br1, br2 = map(int, input().split())
if (br1 != br2) :
    print("\nJumlah tidak sama")
else :
    isi1 = list(map(int, input().split()))
    isi2 = list(map(int, input().split()))
    a1 = []; a2 = []
    for a in range(0, br1) :
        a1.append(isi1[a])
    for a in range(0, br2) :
        a2.append(isi2[a])
    print ()
    for a in range(0, br1) :
        print(a1[a]*a2[a], end=' ')

File: PRAK603-2210817210023-MuhammadShofyAkmal.c
#include <stdio.h>
int main () {
int br1, br2, a, isi;
scanf("%d %d", &br1, &br2);
int a1[br1], a2[br2];
if (br1!=br2) {
    printf ("\nJumlah tidak sama");}
else {
    for (a=0; a<br1; a++) {
        scanf ("%d", &isi);
        a1[a]=isi;}
    for (a = 0; a<br2; a++) {
        scanf ("%d", &isi);
        a2[a]=isi;}
    printf ("\n");
    for (a=0; a<br1; a++) {
        printf ("%d ", a1[a]*a2[a]);}}
}